testapp/src/main/java/com/rakuten/tech/mobile/testapp/helper/UUIDHelper.kt

```diff
@@ -1,11 +1,11 @@
 package com.rakuten.tech.mobile.testapp.helper
 
-import java.lang.Exception
-import java.util.*
+import java.lang.IllegalArgumentException
+import java.util.UUID
 
 fun String.isInvalidUuid(): Boolean = try {
     UUID.fromString(this)
     false
-} catch (e: Exception) {
+} catch (e: IllegalArgumentException) {
```

Comment:
What will be the other case when this fails?

Reply:
Currently, `UUID.fromString(this)` is designed to throw only `IllegalArgumentException`. So, it's enough to use this specific exception. But, we should add the related exception in future when the implementation will be changed inside `String.isInvalidUuid()` function.

Reply:
Currently, `UUID.fromString(this)` is designed to throw `IllegalArgumentException`. So, it's enough to use this specific exception now. But, we should add the related exception in future when the implementation will be changed inside `String.isInvalidUuid()` function.